apps/server/lib/bootstrap.ts

```diff
@@ -275,6 +275,147 @@ export const bootstrap = async (logContext: LogContext, options: any) => {
 		}),
 	);
 
+	await worker.replaceCard(
+		logContext,
+		kernel.adminSession()!,
+		worker.typeContracts['type@1.0.0'],
+		{
+			attachEvents: false,
+			timestamp: new Date().toISOString(),
+		},
+		{
+			slug: 'flowdock-message',
+			type: 'type@1.0.0',
+			name: 'Flowdock message',
+			markers: [],
+			data: {
+				schema: {
+					required: ['data'],
+					type: 'object',
+					properties: {
+						data: {
+							required: [
+								'username',
+								'userFullName',
+								'flowdockUserId',
+								'flowdockUUID',
+								'sent',
+							],
+							type: 'object',
+							properties: {
+								flowdockUUID: {
+									type: 'string',
+								},
+								flowdockUserId: {
+									type: 'string',
+								},
+								userFullName: {
+									type: 'string',
+								},
+								username: {
+									type: 'string',
+								},
+								content: {
+									type: 'string',
+									format: 'markdown',
+									fullTextSearch: true,
+								},
+								sent: {
+									type: 'string',
+									format: 'date-time',
+								},
+								file: {
```

Comment:
@LucianBuzzo I might be doing something wrong here, but files uploaded here are not being returned. Not sure what I am doing wrong.

Reply:
see here in the upload script https://github.com/balena-io-playground/flowdock-to-jf-uploader/blob/master/index.js#L90-L95

maybe i am doing something wrong there.